precompiles/assets-erc20/src/lib.rs

```diff
@@ -413,8 +410,6 @@ where
 		context: &Context,
 	) -> EvmResult<PrecompileOutput> {
 		let mut gasometer = Gasometer::new(target_gas);
-		gasometer.record_cost(RuntimeHelper::<Runtime>::db_read_gas_cost())?;
-		gasometer.record_cost(RuntimeHelper::<Runtime>::db_write_gas_cost())?;
```

Comment:
Why this?

Reply:
They were wrong. I think they were left-overs from when we were using a local approval storage and not the pallet one. Now the db writes assocaited with the pallet-storage should be returned by the dispatch call itself